readthedocsext/theme/templates/profiles/private/migrate_to_gh_app.html

```diff
@@ -0,0 +1,303 @@
+                <div class="title">{% trans "Migrate projects" %}</div>
+                <div class="description">
+                  {% trans "Migrate your projects to the GitHub App" %}
+                </div>
+              </div>
+            </a>
+            <a class="step {% if step == "revoke" %}active{% endif %} {% if not step_connect_completed %}disabled{% endif %}"
+               href="{% url 'migrate_to_github_app' %}?step=revoke">
+              <div class="content">
+                <div class="title">{% trans "Revoke access to old integration" %}</div>
+                <div class="description">
+                  {% trans "Revoke access to the old GitHub OAuth app from your GitHub account" %}
+                </div>
+              </div>
+            </a>
+            <a class="step {% if step == "disconnect" %}active{% endif %} {% if step_disconnect_completed %}completed{% endif %} {% if not step_connect_completed or not step_revoke_completed %}disabled{% endif %}"
+               href="{% url 'migrate_to_github_app' %}?step=disconnect">
+              <div class="content">
+                <div class="title">{% trans "Disconnect old integration" %}</div>
+                <div class="description">
+                  {% trans "Disconnect the old GitHub OAuth app from your Read the Docs account" %}
+                </div>
+              </div>
+            </a>
+          </div>
+        </div>
+        <div class="eleven wide computer eleven wide tablet sixteen wide mobile column">
+          <div class="ui basic segment">
+            {% if step == "overview" %}
+              <p>
+                {% blocktrans trimmed with blog_post="#" %}
+                  We’re introducing a new GitHub App to replace the old GitHub OAuth app.
+                  This new app offers more granular permissions and better GitHub integration.
+                  Learn more in our <a href="{{ blog_post }}">blog post</a>.
+                {% endblocktrans %}
+              </p>
+
+              {% if has_multiple_github_accounts %}
+                <div class="ui message warning">
+                  <i class="fad fa-warning icon"></i>
+                  {% url 'socialaccount_connections' as socialaccount_connections %}
+                  {% blocktrans trimmed with socialaccount_connections=socialaccount_connections github_account=old_github_account.extra_data.login %}
+                    You have <a href="{{ socialaccount_connections }}">multiple GitHub accounts</a> linked,
+                    <strong>you'll need to repeat the migration for each account</strong>.
+
+                    The current account to migrate is <a href="https://github.com/{{ github_account }}" target="_blank">{{ github_account }}</a>;
+                    you can skip repositories this account doesn't have access to.
+                  {% endblocktrans %}
+                </div>
+              {% endif %}
+
+              <p>
+                {% blocktrans %}
```

Comment:
You need trimmed on all these blocktrans:

```suggestion
                {% blocktrans trimmed %}
```